cloudify_agent/api/pm/initd.py

```diff
@@ -59,16 +59,14 @@ def configure(self):
 
         self._logger.debug('Creating daemon script: {0}'
                            .format(self.script_path))
-        self._create_script()
+        self.create_script()
```

Comment:
except for the `if self.start_on_boot` part, is this method identical to its super class?
if so, remove all shared code and call `super(GenericLinuxDaemon, self).configure()`